justfile

```diff
@@ -40,11 +41,10 @@ DC_USER := env_var_or_default("DC_USER", "opener")
 
 # Install all dependencies
 @install:
-    just api/install
-    just ingestion_server/install
```

Comment:
Do you think we could leave these in? I find it helpful for local development to have interpreters set up with the necessary dependencies for each project. Although, I suppose this isn't removing the recipes, just removing the need to install them as part of the top level `just install`?

Reply:
Yes, this only removes them from the top-level install because they're needed in specific only in specific cases, not always, and we recommend using Docker for most local development.